public/js/controllers/index.js

```diff
@@ -166,10 +178,23 @@ angular.module('mean.system').controller('IndexController', [
       messageArray.push(message);
       $scope.notifications = messageArray;
       $scope.messageLength = messageArray.length;
+      console.log('@@@@@@@@@@@',messageArray);
     });
 
-    window.onload = () => {
-      $scope.userGames()
+    $scope.tab2 = false;
+    $scope.tab1 = true;
+    $scope.inviteModal = false;
+    $scope.showtab1 = () => {
```

Comment:
The feedback by @chuzksy-codeactive should also apply here. These names (_showtab1_, _showtab2_) are not descriptive enough based on their essence. I suggest you use variable/parameter names that are descriptive.

Reply:
I have implemented it